models/engine/file_storage.py

```diff
@@ -75,5 +94,6 @@ def get(self, cls, id):
                 if type(cls) == str and type(id) == str else None
 
     def count(self, cls=None):
-        """Count number of objects in storage or specific number of cls objects"""
+        """Count number of objects in storage or specific number
+        of cls objects"""
         return len(self.all(cls))
```

Comment:
Same comment as before in db_storage.py. "Very compact code, but it could also use the sqlalchemy code has a count() method."